### PR TITLE
MAT-8680: Update SR logic.

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-8680](https://jira.cms.gov/browse/MAT-8680)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/components/cqlLibraryList/CqlLibraryList.tsx [hunk 1]
```diff
@@ -526,7 +531,16 @@ export default function CqlLibraryList({
             style={{ borderColor: "#c8c8c8" }}
             onClick={(e) => handleOpen(info.row.original, e)}
             data-testid={`view/edit-cqlLibrary-button-${info.row.original.id}`}
-            aria-label={`CQL Library ${info.row.original.cqlLibraryName} version ${info.row.original.version} draft status ${info.row.original.draft} View / Edit`}
+            aria-label={`${
+              checkUserCanEdit(
+                info.row.original.librarySet?.owner,
+                info.row.original.librarySet?.acls
+              ) && info.row.original.draft
```
Comment (adongare) [inline src/components/cqlLibraryList/CqlLibraryList.tsx:533]:
```suggestion
              canEdit && info.row.original.draft
```

File: src/components/cqlLibraryList/CqlLibraryList.tsx [hunk 2]
```diff
@@ -548,7 +562,16 @@ export default function CqlLibraryList({
                 ? `edit-cql-library-button-${info.row.original.id}`
                 : `view-cql-library-button-${info.row.original.id}`
             }
-            aria-label={`CQL Library ${info.row.original.cqlLibraryName} version ${info.row.original.version} draft status ${info.row.original.draft} View / Edit`}
+            aria-label={`${
+              checkUserCanEdit(
+                info.row.original.librarySet?.owner,
+                info.row.original.librarySet?.acls
+              ) && info.row.original.draft
```
Comment (adongare) [inline src/components/cqlLibraryList/CqlLibraryList.tsx:564]:
```suggestion
              canEdit && info.row.original.draft
```

File: src/components/cqlLibraryList/CqlLibraryList.tsx [hunk 3]
```diff
@@ -548,7 +562,16 @@ export default function CqlLibraryList({
...
+                ? `Edit`
+                : `View`
+            } Library ${info.row.original.cqlLibraryName} ${
+              info.row.original.version
+            }${info.row.original.draft ? " Draft" : ""}`}
           >
             {checkUserCanEdit(
```
Comment (adongare) [inline src/components/cqlLibraryList/CqlLibraryList.tsx:571]: although its existing, can be updated quickly...same here

**Review by adongare — APPROVED** [review]
